db/migrate/20180403144853_convert_vm_attrs_to_hash.rb

```diff
@@ -0,0 +1,80 @@
+class ConvertVmAttrsToHash < ActiveRecord::Migration[5.1]
+  def up
+    transform :to_h
+  end
+
+  def down
+    transform :to_params
```

Comment:
do we actually want this? i don't think we want the AC:Params stored in db anyways tbh